app/views/layouts/_flash_message.html.erb

```diff
@@ -1,3 +1,5 @@
+<%# locals: (message: nil) %>
```

Comment:
I think this partial should always receive a `message`, so I suggest removing `nil` and raising an error if this local isn't present
```suggestion
<%# locals: (message:) %>
```